Formula/telnet.rb

```diff
@@ -0,0 +1,46 @@
+class Telnet < Formula
+  desc "Apple telnet built from macOS 10.12.4 sources"
```

Comment:
> I also don't think there's anything wrong with offering multiple telnet clients such as inetutils and the FreeBSD version alongside the Apple version.

I've looked at several man pages, and they all have quite similar `telnet` description. Shouldn't formula description indicate precisely differences from other hypothetical formulas? Or, we may consider using both descriptions (something like `User interface to the TELNET protocol (built from macOS 10.12.4 source)`)?

https://linux.die.net/man/1/telnet
http://www-it.desy.de/cgi-bin/man-cgi?telnet+1
https://www.freebsd.org/cgi/man.cgi?telnet